Resources/views/CRUD/list_array.html.twig

```diff
@@ -8,19 +8,17 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% import _self as list %}
-{%  macro render_array(value) %}
-    {% for key, val in value %}
-        {% if val is iterable %}
```

Comment:
Yeah, your are right, it was in lists (started from the show template which didn't handle them). However, it's not working (list should be replaced with _self in the macro).